x-pack/plugins/ml/public/application/components/ml_inference/components/configure_pipeline.tsx

```diff
@@ -0,0 +1,213 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC, memo } from 'react';
+
+import {
+  EuiCode,
+  EuiFieldText,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiForm,
+  EuiFormRow,
+  EuiLink,
+  EuiSpacer,
+  EuiTitle,
+  EuiText,
+  EuiTextArea,
+  EuiPanel,
+} from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useMlKibana } from '../../../contexts/kibana';
+import type { MlInferenceState } from '../types';
+
+interface Props {
+  handlePipelineConfigUpdate: (configUpdate: Partial<MlInferenceState>) => void;
+  modelId: string;
+  pipelineNameError: string | undefined;
+  pipelineName: string;
+  pipelineDescription: string;
+  targetField: string;
+  targetFieldError: string | undefined;
+}
+
+export const ConfigurePipeline: FC<Props> = memo(
+  ({
+    handlePipelineConfigUpdate,
+    modelId,
+    pipelineName,
+    pipelineNameError,
+    pipelineDescription,
+    targetField,
+    targetFieldError,
+  }) => {
+    const {
+      services: {
+        docLinks: { links },
+      },
+    } = useMlKibana();
+
+    const handleConfigChange = (value: string, type: string) => {
+      handlePipelineConfigUpdate({ [type]: value });
+    };
+
+    return (
+      <EuiFlexGroup>
+        <EuiFlexItem grow={3}>
+          <EuiTitle size="s">
+            <h4>
+              {i18n.translate(
+                'xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.configure.title',
+                { defaultMessage: 'Create a pipeline' }
+              )}
+            </h4>
+          </EuiTitle>
+          <EuiSpacer size="m" />
+          <EuiText color="subdued" size="s">
+            <p>
+              <FormattedMessage
+                id="xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.configure.description"
+                defaultMessage="Build a {pipeline} to use the trained data frame analytics model - {modelId} - for inference."
+                values={{
+                  modelId: <EuiCode>{modelId}</EuiCode>,
+                  pipeline: (
+                    <EuiLink external target="_blank" href={links.ingest.pipelines}>
+                      pipeline
+                    </EuiLink>
+                  ),
+                }}
+              />
+            </p>
+            <p>
+              {i18n.translate(
+                'xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.configure.descriptionUsePipelines',
+                {
+                  defaultMessage:
+                    "Use pipeline/_simulate or _reindex API to pass data into this pipeline. The model's predictions will be stored in the Target field.",
```

Comment:
```suggestion
                    "Use pipeline/_simulate or _reindex API to pass data into this pipeline. Predictions are stored in the Target field.",
```

Reply:
Updated in 64ba3ac4ec279eb78f5f8a97d01005991ae76219